plugin/clone/include/clone.h

```diff
@@ -90,6 +90,10 @@ extern uint clone_max_io_bandwidth;
 /** Clone system variable: If network compression is enabled */
 extern bool clone_enable_compression;
 
+extern unsigned long clone_compression_algorithm;
```

Comment:
```suggestion
extern ulong clone_compression_algorithm;
```
for consistency with `uint`